nova/src/hash/circuit.rs

```diff
@@ -0,0 +1,105 @@
+use crate::hash::Mimc;
+use r1cs::gadget::curve::PointAssignment;
+use r1cs::gadget::field::FieldAssignment;
+use r1cs::{CircuitDriver, R1cs};
+use zkstd::common::IntGroup;
+
+pub(crate) struct ZKMimc<const ROUND: usize, C: CircuitDriver> {
```

Comment:
Considering name rule so far, MimcAssignment is fine

Reply:
Renamed